module.modulemap

```diff
@@ -0,0 +1,12 @@
+// This file is required by current CocoaPods (0.39.0). 
+//  - Generates warning: Umbrella header for module 'SQLite' does not include header 'fts3_tokenizer.h'
```

Comment:
I tried all sorts of things. CocoaPods is a magical mess was the only conclusion I could come up with. I essentially had to choose between keeping Xcode happy, keeping CocoaPods happy, or crawling under the bed to cry. I hoped that once this PR went "live", someone would point out the stupidity of my ways and offer a fix!

That said, I've somewhat recovered from the trauma, and will see if making it private makes the problem go away! ;)